package.json

```diff
@@ -10,11 +10,11 @@
     "prepublish": "npm run build",
     "lint": "eslint --ext .js,.jsx src test stories",
     "storybook": "start-storybook -p 9001",
-    "pretest": "npm run lint",
-    "test": "npm run build && npm run test:coverage",
-    "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- -R tap 'test/**/*-test.*'",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/*-test.*",
+    "pretest": "npm run build && npm run lint",
+    "test": "nyc -e .jsx --reporter=lcov --reporter=text npm run test:unit",
```

Comment:
aha, thanks for clarifying; that wasn't clear to me.

it might help to have two separate commands, one to run with dom, and one without, and then have a test script that runs both?